README.md

```diff
@@ -29,19 +29,25 @@ HexChat developers decided that their script should focus on their specific need
     pacman -S nasm patch gettext make coreutils diffutils wget yasm pkg-config
     ```
 
-1. Clone [this repository](https://github.com/wingtk/gtk-win32) to _C:\gtk-build\github\gtk-win32_ It contains the build script, project files and patches.
+1. Clone [this repository](https://github.com/wingtk/gvsbuild) to _C:\gtk-build\gvsbuild_ It contains the build script, project files and patches.

```

Comment:
I agree they should be precise but currently there is no need to create the github directory. So the instructions are:
git clone repo
cd repo
python build.py build gtk3